src/docs/metrics/nft-social-volume/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: NFT Social Volume
+author: Ilya
+date: 2022-05-15
+description: The amount of messages/documents containing a mention of NFT Collection
+# REF metrics-hub/metricshub/nft_collections_social_volume.py
+---
+
+## Definition
+
+Social Volume is build on top of the [Social Data](/metrics/details/social-data).
+The idea is pretty much the same as for [Social Volume](/metrics/social-volume).
+
+The total number of [text documents](/metrics/details/text-document) that
+contain the mention of certain NFT Collection at least one. Examples of documents are telegram
+messages and reddit posts. E.g., if a single short telegram message includes the word
+`bored ape` more than once, this message will increase the social volume of the
+Bored Ape Yacht Club by 1. If a long reddit post contains the word `bored ape` 10 times,
+this again will increase the social volume of the Bored Ape Yacht Club by 1.
+
+Social Volume computed for NFT contract. For each contract we construct a special search term.
+We use machine learning model based on TF-IDF for computing those search terms. The idea is that we try to compute search terms based on pairwise similarity of NFTs' names. For computing embedding per each collections' name we use TF-IDF model on n-grams of chars.
+
+Some search terms examples for some NFT collections:
+- [CryptoPunks](https://www.larvalabs.com/cryptopunks) - `cryptopunks`;
+- [Veefriends Series 2](https://series2.veefriends.com/) - `veefriends`;
+- [CyberKongz VX](https://www.cyberkongz.com/) - `cyberkongz`.
+
+---
+
+## Access
+
+[Restricted Access](/metrics/details/access#restricted-access).
+
+---
+
+## Measuring Unit
+
+Amount of documents that mention the given NFT Collection.
```

Comment:
Btw do we use the same words in Social Volume Measuring Unit section? @ilya-l-san

Reply:
Social Volume Measuring Unit is very close: `"Amount of documents that mention the given text pattern."`